lib/matplotlib/tests/test_backend_ps.py

```diff
@@ -256,6 +256,17 @@ def test_linedash():
     assert buf.tell() > 0
 
 
+def test_empty_line():
+    # Smoke-test for gh#23954
+    figure = Figure()
+    ax = figure.add_subplot(111)
+    ax.set_title('\nLower title')
+    ax.annotate(text='\nLower label', xy=(0, 0))
```

Comment:
Would a figtext (fig.text(...), no axes at all) be enough?

Reply:
It is. I've changed it.